definitions/apm-application/definition.yml

```diff
@@ -11,3 +11,22 @@ compositeMetrics:
 configuration:
   entityExpirationTime: QUARTERLY
   alertable: true
+
+relationships:
+  # Select entityGuid, peer.hostname from Span where peer.hostname is not null and category =
+  # 'datastore'
+  # ExtService in this case is a DB
+  - provider: newrelic
+    eventType: Span
+    source:
+      guid: entityGuid
```

Comment:
I'd assume that relationship synthesis would happen after entity synthesis, when the `entity.guid` is already set (either by synthesis, or by another ingestion service).  Taking that into account, could we just remove `source` altogether? (e.g. assuming the guid is at `entity.guid`).

Reply:
That said, we most likely want to filter on  specific events (e.g. "apply this rule when the event is of type EXT-SERVICE). But I think the guid field should be taken by convention.

Reply:
my original draft was as you said, but then while defining infra host HOSTS X entity, I realised I wanted to change direction of the relationship i was creating. 

Saying that, we could still take the guid field by convention and force to specify only direction

Reply:
done!